docs/data/stashcache/install-cache.md

```diff
@@ -108,89 +138,84 @@ and runs alongside the unauthenticated instance.
 
-### Add Authfile for authenticated cache
-
-The Authfile for the authenticated cache is located in `/etc/xrootd/Authfile-auth`.
-This is a separate file from the non-authenticated cache Authfile.
 
-Since the authenticated cache runs alongside the unauthenticated cache,
-care must be taken to avoid conflicts between the two.
-In particular, paths that are accessible via the authenticated cache should not be accessible via the unauthenticated cache,
-and vice versa.
+### Add a service certificate
 
-As an example:
+The service will need a certificate to authenticate to StashCache origins.
+Do the following:
 
-```console
-root@host # cat /etc/xrootd/Authfile-auth
-g /osg/ligo /user/ligo rl
-u ligo /user/ligo rl
-```
+1. Copy the host certificate to `/etc/grid-security/xrd/xrd{cert,key}.pem`
+1. Set the owner of the directory and contents `/etc/grid-security/xrd/` to `xrootd:xrootd`:
 
-This permits users in the VOMS group `/osg/ligo` and users mapped to `ligo` to read and list anything under
-`/user/ligo`.
+        :::console
+        root@host # chown -R xrootd:xrootd /etc/grid-security/xrd/
 
-When ready with configuration, you may [start](#managing-stashcache-and-associated-services) your Cache server.
+When ready with configuration, you may [start](#managing-stashcache-and-associated-services) your xrootd@stash-cache-auth service.
 
 Configuring Optional Features
 -----------------------------
 
 ### Adjust disk utilization
 
-To adjust the disk utilization of your StashCache cache, modify the values of `pfc.diskusage` in `/etc/xrootd/xrootd-stashcache-cache-server.cfg`:
+To adjust the disk utilization of your StashCache cache, create or edit a file named `/etc/xrootd/config.d/90-local.cfg`
+and set the values of `pfc.diskusage`.
 
 ```
-pfc.diskusage 0.98 0.99
+pfc.diskusage 0.90 0.95
 ```
 
-The first value and second values correspond to the low and high usage watermarks, respectively, in fractions. When the high watermark is reached, the XRootD service will automatically purge cache objects down to the low watermark.
+The two values correspond to the low and high usage water marks, respectively.
+When usage goes above the high water mark,
+the XRootD service will delete cached files until usage goes below the low water mark.
+
 
 ### Enable remote debugging
 
-This feature enables remote debugging via the `digFS` read-only file system, it's optional line in the config file that was created when configuring the cache:
+XRootD provides remote debugging via a read-only file system named digFS.
+This feature is disabled by default, but you can enable it when you need assistance with your server.
 
-```
-xrootd.diglib * /etc/xrootd/digauth.cf
-```
+To enable remote debugging, edit `/etc/xrootd/digauth.cfg` and specify the authorizations for reading digFS.
 
-where `/etc/xrootd/digauth.cf` may have following content:
+See [the XRootD manual](http://xrootd.org/doc/dev48/xrd_config.htm#_Toc496911334) for the syntax.
```

Comment:
Can we give an example at least? This seems pretty user unfriendly. Also, do the XRootD services need to be restarted for this to take effect?